alteryx/featuretools
# v1.12.1

v1.12.1 Aug 4, 2022
===================
* Fixes
    * Update ``Trend`` and ``RollingTrend`` primitives to work with ``IntegerNullable`` inputs (#2204)
    * ``camel_and_title_to_snake`` handles snake case strings with numbers (#2220)
    * Change ``_get_description`` to split on blank lines to avoid truncating primitive descriptions (#2219)
* Documentation Changes
    * Add instructions to add new users to featuretools feedstock (#2215)
* Testing Changes
    * Add create feedstock PR workflow (#2181)
    * Add performance tests for python 3.9 and 3.10 (#2198, #2208)
    * Add test to ensure primitive docstrings use standardized verbs (#2200)
    * Configure codecov to avoid premature PR comments (#2209)
